.github/workflows/ci.yml

```
@@ -0,0 +1,231 @@
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.13"
+
+    - name: Install dependencies
+      run: uv pip install -e ".[dev]"
+
+    - name: Lint with Ruff
+      run: uv run ruff check .
+
+    - name: Format check with Ruff  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)